pkg/band/ism_2400.go

```diff
@@ -42,7 +42,7 @@ func init() {
 			MaxDataRate: ttnpb.DATA_RATE_7,
 		},
 	}
-	const beaconFrequency = 2422000000
+	const beaconFrequency = 2424000000
```

Comment:
What is the source for this? It's not in https://github.com/TheThingsNetwork/lorawan-stack/issues/3067

cc @Oliv4945

Reply:
This is from the v4 draft, please see the spec.